tm2py/components/network/transit/transit.py

```diff
@@ -1596,6 +1659,52 @@ def trim_demands(self, scenario, period):
                 ]
                 matrix_calc(spec_list, scenario=scenario, num_processors=num_processors)
 
+    def export_boardings_by_station(self, network, use_fares, period, scenario):
```

Comment:
Does this currently only export the boardings for BART and Caltrain?

Reply:
https://github.com/camsys/tm2py/blob/5d769ce1e55b18003a22977f04bfb18d61666adf/tm2py/components/network/transit/transit.py#L1683-L1692

Reply:
Yes, it’s currently set only for these two operators (modes).